config/initializers/gitlab_shell_secret_token.rb

```diff
@@ -14,6 +13,8 @@
   File.write(secret_file, token)
 end
 
-if File.exist?(Gitlab.config.gitlab_shell.path) && !File.exist?(gitlab_shell_symlink)
-  FileUtils.symlink(secret_file, gitlab_shell_symlink)
+# TODO May be removed after updating the default secret_file path...?
```

Comment:
When you set the same secret_path in gitlab and gitlab-shell, then there’s no need for symlink.